src/olympia/yara/tests/test_admin.py

```diff
@@ -0,0 +1,94 @@
+import json
+
+from django.contrib.admin.sites import AdminSite
+from django.utils.html import format_html
+
+from olympia import amo
+from olympia.amo.tests import (TestCase, addon_factory, user_factory,
+                               version_factory)
+from olympia.amo.urlresolvers import reverse
+from olympia.yara.admin import YaraResultAdmin
+from olympia.yara.models import YaraResult
+
+
+class TestYaraResultAdmin(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = user_factory()
+        self.grant_permission(self.user, 'Admin:Advanced')
+        self.client.login(email=self.user.email)
+        self.list_url = reverse('admin:yara_yararesult_changelist')
+
+        self.admin = YaraResultAdmin(model=YaraResult, admin_site=AdminSite())
+
+    def test_list_view(self):
+        response = self.client.get(self.list_url)
+        assert response.status_code == 200
+
```

Comment:
I like to add a simple test proving that without the specific permission (but an `Admin:<something>` permission left so that the user can still access django admin itself), `self.client.get(self.list_url)` returns a 403.

Reply:
Good call, thanks!